src/includes/performance/enable-automatic-instrumentation/javascript.mdx

```diff
@@ -1,4 +1,4 @@
-After configuration, you will see both `pageload` and `navigation` when viewing transactions in sentry.io.
+After configuration, you will see both `pageload` and `navigation` transactions in sentry.io.
```

Comment:
Small fix to formatting of sentry.io so that it's a link

```suggestion
After configuration, you will see both `pageload` and `navigation` transactions in [sentry.io](https://sentry.io).
```

Reply:
Not really meant to be a link - more meant to be talking about it as a general idea. Lemme change to "the Sentry UI."